analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -349,11 +349,22 @@ class Pip(
 
         val jsonFile = createOrtTempDir().resolve("python-inspector.json")
 
+        val pythonVersion = when (pythonMajorVersion) {
+            2 -> "2.7" // 2.7 is the only 2.x version supported by python-inspector.
+            3 -> "3.10" // 3.10 is the version currently used in the ORT Docker image.
```

Comment:
Just an idea that came to my mind: Could we make this code more generic by running "python2 --version" / "python3 --version" instead of hard-coding the versions here? That way, we'd not need to update this code if the Python versions in the Docker image change.

Reply:
Yes, I will do that in the next PR, but first I want to do some other test improvements.